texar/modules/connectors/connectors.py

```diff
@@ -438,29 +467,32 @@ class ReparameterizedStochasticConnector(ConnectorBase):
 
         .. code-block:: python
 
+            # Initialized without num_samples
             cell = LSTMCell(num_units=256)
             # cell.state_size == LSTMStateTuple(c=256, h=256)
-            connector = ReparameterizedStochasticConnector(cell.state_size)
-            kwargs = {
-                'loc': torch.zeros([batch_size, 10]),
-                'scale_diag': torch.stack(
-                    [torch.diag(x) for x in torch.ones([batch_size, 10])],
-                    0)
-            }
-            output, sample = connector(distribution_kwargs=kwargs)
-            # output == LSTMStateTuple(c=tensor_of_shape_(batch_size, 256),
-            #                          h=tensor_of_shape_(batch_size, 256))
-            # sample == Tensor([batch_size, 10])
-            kwargs = {
-                'loc': torch.zeros([10]),
-                'scale_diag': torch.stack(
-                    [torch.diag(x) for x in torch.ones([10])], dim=0)
-            }
+            mu = torch.zeros([16, 100])
+            var = torch.ones([16, 100])
+            #var = torch.stack([torch.diag(x) for x in var], 0)
+            f_mu = torch.flatten(mu)
+            f_var = torch.flatten(var)
+            gauss_ds = tds.multivariate_normal.MultivariateNormal(
+                loc=f_mu,
+                scale_tril=torch.diag(f_var))
+            connector = ReparameterizedStochasticConnector(
+                cell.state_size, distribution=gauss_ds)
+            output, sample = connector()
+            # output == LSTMStateTuple(c=tensor_of_shape_(1, 256),
+            #                          h=tensor_of_shape_(1, 256))
+            # sample == Tensor([16 * 100])
+
+            # Initialized with num_samples
+            connector = ReparameterizedStochasticConnector(
+                cell.state_size, distribution=gauss_ds, num_samples=4)
             output_, sample_ = connector(distribution_kwargs=kwargs,
```

Comment:
Please update this code example. I don't think we have `distribution_kwargs` anymore.